### PR TITLE
Chrome extension update.

### PR DESCRIPTION
**Pull request by PrashanthAmbure**
Chrome extension core logic hasn't been disturbed on this commit instead, a gulp task has been added to create a crx file with signed zip file.

### CODE REVIEW

File: plugins/chrome/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "KodeBeagle",
+  "version": "0.1.1",
+  "description": "KodeBeagle Chrome extension. An extension that makes searching selected piece of java code easily searcheable on KodeBeagle.",
```
Comment (ScrapCodes) [inline plugins/chrome/package.json]: >KodeBeagle Chrome extension. An extension that makes searching selected piece of java code easily searcheable on KodeBeagle.

_can be changed to:_
"Select some lines of java code on the browser and right click to search it for more similar code snippets on kodebeagle.com"
@minidoc is this better ?

File: plugins/chrome/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  ],
+  "author": "Pramati",
+  "contributors": [
+      {
+        "name":"Prashanth Ambure",
+        "email":"prashanth.ambure@imaginea.com"
+      }
+  ],
+  "private": true,
+  "license": "MIT"
```
Comment (ScrapCodes) [inline plugins/chrome/package.json]: license, I am not sure. But we use Apache 2 for rest of the project.

File: plugins/chrome/src/background.js [hunk 3]
```diff
@@ -1,63 +1,63 @@
...
-
-chrome.contextMenus.create({
-  title: contextMenuTitle,
-  contexts: ["selection"],
-  onclick: evalFunction
-});
-
-// We don't want the search to consider java.lang classes. 
-// This object will be initialized with external JSON file named java_lang_pkg.json
-// This contains all the elements which would be eliminated.
```
Comment (ScrapCodes) [inline plugins/chrome/src/background.js]: What is changed in these lines ?

File: plugins/chrome/package.json (hunk 1 above)
Comment (ScrapCodes) [inline plugins/chrome/package.json]: This also applies to description in manifest file.

File: plugins/chrome/src/background.js (hunk 3 above)
Comment (PrashanthAmbure) [inline plugins/chrome/src/background.js]: The change is this file along with two other files which are required for crx file are moved to a src folder.

File: plugins/chrome/README.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+![](https://lh3.googleusercontent.com/fF8uuEASxzulF94sLk8uhNVpvkaQUIBD-GdxPgucC-5smOSWh25oCPeNpMlVymgPZeudRTAz-G7bDbQ_4OSySTbwOKeNKIJ7mrvJXWsMZenitsiHvoYSBl69h85mRd3933AgR51QB2Rskj1yTPruiJe7i1a3IyEo68ASW5czsKfWYv1kta3xgURh098fYk8lfCXU140ye1Jb7vcLAsKJZK272RJEXiMDQjq8qHHlhk5fq-GQ2l1OT4-3e3YX1wjd1OckLK8-QZvUQzrsd0BIQaHbPaQ-O_Ry4q6hBERcx32OXi1sMzx_8RmxRDLOVzBqMrFp18nX-RUYy6rFKwAt0qTBZnnn4BVl7tZ3s2fCZwF0qzTSZi86FCcLjrRrgVijWt1_NCquhuuzBV-DsLr5Hgbdd4zfR_tqJpKtw1eZ_z-Ln7_LqpdO9ZKlpsPHfkt2H94jIY16G6mypV1g6o12QDdQrKk3cE396T3p2m3FAiwqjuS0Ki90oA1hbOP-hs4HxWeDu6J2wioj7mD1RMifKoBN-sJTthBDSazXpWbKzDI=w1227-h655-no)
+
+## Developer notes
+### Requirements
+* [NodeJS & NPM](http://nodejs.org/download)
+* _gulp_ node package
+* chrome browser
+
+### Loading unpacked extension.
+1. Open the chrome browser and type in - _**chrome://extensions**_.
```
Comment (ScrapCodes) [inline plugins/chrome/README.md]: I can not see these images on my system.

Comment (PrashanthAmbure) [inline plugins/chrome/README.md]: @ScrapCodes Could you please verify now?

Comment (minidoc) [inline plugins/chrome/README.md]: Please change the typo "windown".

File: plugins/chrome/package.json (hunk 1 above)
Comment (ScrapCodes) [inline plugins/chrome/package.json]: @PrashanthAmbure and @minidoc Any comment about this improvement for description ?

Comment (minidoc) [inline plugins/chrome/package.json]: Sorry missed your note from earlier. 
Yes, the description needs to be changed. I like the one you put @ScrapCodes . 

Another suggestion is "Quick access to search on kodebeagle.com for any Java code snippet you select from the browser."

File: plugins/chrome/README.md (hunk 4 above)
Comment (minidoc) [inline plugins/chrome/README.md]: Could you also add hyperlinks for the ToC? Follow https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet as example.

File: plugins/chrome/package.json (hunk 1 above)
Comment (ScrapCodes) [inline plugins/chrome/package.json]: "Quick access to code samples(in Java) from github by searching on kodebeagle.com".

How to use ?
(Steps)

**Comment by ScrapCodes** [comment]
@minidoc Please take a look !, Can I merge this ?

**Pull request closed without merging**